validation/validation_service.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import json
+import os
+
+from .not_known_entity_exception import NotKnownEntityException
+from .validation_state import ValidationState
+
+
+class ValidationService:
+    SCHEMA_FILENAME_PREFIX = "covid_data_uploader-"
+    SCHEMA_FILENAME_EXTENSION = ".json"
+    SCHEMA_FILES_FOLDER = "../json-schema/"
+    ENTITY_MAPPING_FILE = "../config/schema_by_entity_mapping.json"
```

Comment:
this file should just be in the root of the /validation package.

or better, store the file names as `study.json`, `sample.json`, `run_experiment.json` and `isolate_genome_assembly_information.json` so that we don't have to map at all.

Reply:
We can come back to this as part of a refactor pass.